scripts/zones/Attohwa_Chasm/Zone.lua

```diff
@@ -85,6 +85,21 @@ zone_object.onGameHour = function(zone)
         starting at ID.npc.MIASMA_OFFSET. some are supposed to toggle open, but need retail test
         to determine which.  for now, they're just statically set per npc_list.animation
     --]]
+
+    -- Don't allow Citipati or Xolotl to spawn outside of night
+    local xolre = GetMobByID(ID.mob.XOLOTL):getLocalVar("xolotlRespawn")
+
+    if VanadielHour() >= 4 and VanadielHour() < 20 then
```

Comment:
I might be wrong, but I believe the nature of its other mechanics precludes the use of the spawn type that would normally handle it and would cause it to then pop every night. worth double checking.